rc/tools/git.kak

```diff
@@ -163,14 +163,14 @@ define-command -params .. -docstring %{
 define-command -params .. -docstring %{
     git-diff [<file>]...: show the diff of the given files
 } git-diff %{ evaluate-commands %sh{
-    output=$(mktemp -d "${TMPDIR:-/tmp}"/kak-git-diff.XXXXXXXX)/fifo
+    output=$(mktemp "${TMPDIR:-/tmp}"/kak-git-diff.XXXXXXXX)
     mkfifo "${output}"
```

Comment:
This leads to an error like
```
mkfifo: cannot create fifo '/tmp/kak-git-diff.kzg6uxub': File exists
```

Reply:
Thanks for testing and sorry about that, I messed up my `git add -p`. It should be fixed now, I added the appropriate `rm "${output}"` call.